open-api/rest-catalog-open-api.yaml

```diff
@@ -770,6 +770,52 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/{prefix}/namespaces/{namespace}/tables/{table}/metrics:
+    parameters:
+      - $ref: '#/components/parameters/prefix'
+      - $ref: '#/components/parameters/namespace'
+      - $ref: '#/components/parameters/table'
+
+    post:
+      tags:
+        - Catalog API
+      summary: Send a scan report to this endpoint to be processed by the backend
+      description:
+        Send a scan report to this endpoint to be processed by the backend
+      operationId: sendMetrics
+      requestBody:
+        description: The request containing the scan report to be sent
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MetricsRequest'
+        required: true
+      responses:
+        204:
+          description: Success, no content
+        400:
+          $ref: '#/components/responses/BadRequestErrorResponse'
+        401:
+          $ref: '#/components/responses/UnauthorizedResponse'
+        403:
+          $ref: '#/components/responses/ForbiddenResponse'
+        404:
```

Comment:
Do we want a 406 if the service doesn't accept scan reports?

Reply:
that's a good question. I had a quick look and I'm not sure 406 is accurate here:
```
The HyperText Transfer Protocol (HTTP) 406 Not Acceptable client error response code indicates that the server cannot produce a response matching the list of acceptable values defined in the request's proactive content negotiation headers, and that the server is unwilling to supply a default representation.
```